sros2/xml_cache/xml.xsd

```diff
@@ -0,0 +1,117 @@
+<?xml version='1.0'?>
+<!DOCTYPE xs:schema PUBLIC "-//W3C//DTD XMLSCHEMA 200102//EN" "XMLSchema.dtd" >
+<xs:schema targetNamespace="http://www.w3.org/XML/1998/namespace" xmlns:xs="http://www.w3.org/2001/XMLSchema" xml:lang="en">
```

Comment:
Does this need to be cached as well? How far down the rabbit hole do we need to recurse?

Reply:
@ruffsl That line doesn't trigger any network traffic.  Those are just names that happen to be URLs.  They're not URIs that need to be resolved